<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortWidget</class>
 <widget class="QDialog" name="SortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="graderadioButton">
      <property name="text">
       <string>成绩</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="idradioButton">
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sortpushButton">
      <property name="text">
       <string>排序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="returnButton">
      <property name="text">
       <string>主菜单</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>511</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>首次或更新学生信息后需点击&quot;排序&quot;查看信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
